support-frontend/assets/helpers/abTests/__tests__/abtestTest.js

```diff
@@ -39,6 +39,10 @@ describe('basic behaviour of init', () => {
     window.matchMedia = window.matchMedia || jest.fn(() => ({ matches: false }));
   });
 
+  afterEach(() => {
+    window.localStorage.clear();
+  });
+
```

Comment:
Changes to this file are because upgrading to JSDOM 15 broke these tests- the tests were assuming a non-operational localStorage whereas JSDOM 15 has a functional one.